relayer/src/link.rs

```diff
@@ -653,10 +653,6 @@ impl RelayPath {
                 }
                 Err(e) => {
                     // Unrecoverable error, propagate up the stack
-                    error!(
```

Comment:
This error message is redundant.

The log looks like this

> Jul 08 10:52:36.137 ERROR [ibc-1:transfer/channel-0 -> ibc-0] unrecoverable error in send_from_operational_data failed with underlying error: did not find tx confirmation from chain ibc-0 for hash(es) 772CE848125961E520A36A8CCE71171E67F7BBD691EE8E004B021742EEFC1DE6
Jul 08 10:52:36.137 ERROR [ibc-1:transfer/channel-0 -> ibc-0] step() encountered error: failed with underlying error: did not find tx confirmation from chain ibc-0 for hash(es) 772CE848125961E520A36A8CCE71171E67F7BBD691EE8E004B021742EEFC1DE6


so I removed it.